.gitignore

```diff
@@ -378,3 +378,5 @@ android/pytorch_android_torchvision/.cxx
 .arcconfig
 .stable_pyre_client
 .pyre_client
+
+product/    
```

Comment:
What's this for?

Reply:
I used this dir as a temp for out files/dirs /include /lib /bin.
I have removed it.